runtime/compiler/optimizer/J9TransformUtil.cpp

```diff
@@ -155,30 +155,31 @@ TR::Node *
 J9::TransformUtil::generateArrayOffsetTrees(TR::Compilation *comp, TR::Node *indexNode, TR::Node *elementSizeNode, int32_t elementSize, bool useShiftOpCode)
    {
    TR::Node *offsetNode = indexNode->createLongIfNeeded();
+   if (elementSizeNode)
+      elementSizeNode = elementSizeNode->createLongIfNeeded();
+   // TODO_sverma: Need to investigate why using convertStoreDirectToLoadWithI2LIfNeeded
```

Comment:
We can probably delete these lines

Reply:
Comment lines was coming from a merge conflict but can be removed

Reply:
We aren't calling that function anymore so no need to keep the comment around.